### DISCUSSION TITLE
Auto Renaming files and folders

### DISCUSSION THREAD

**Discussion by Tanarion40** [advplyr/audiobookshelf#3178] [Ideas]
Hello.

I was wondering if it would be possible to auto rename folders and files, based on the Metadata of the audiobook.

For example, one my book has Metadata, it rename the file, create folders for author/series or move the file in the correct folder if it exist.
It would make the whole thing much easier when you save thev files

**Comment by nichwall** [comment]
https://github.com/advplyr/audiobookshelf/issues/312
